docs/user/openstack/README.md

```diff
@@ -646,6 +646,24 @@ If you ran the installer with a [custom CA certificate](#self-signed-openstack-c
 oc edit configmap -n openshift-config cloud-provider-config
 ```
 
+### Enabling Octavia for Load Balancer Services
+
+OpenShift supports creating load balancer service types, and ingress controllers with load balancers as a backend using octavia as a day 2 operation. However, there are a few known issues to be aware of: only HTTP, HTTPS, and TCP traffic is supported, and the floating IPs attached to your Octavia load balancers will not be deleted if you delete your cluster.
```

Comment:
I don't really think HTTP or HTTPS is supported either. The code explicitly says only TCP is fine.

This is a bit confusing, I'll try to rephrase it and you can fix my non-native English then. ;)

```
OpenShift can implement LoadBalancer Services (and ingress controllers) using Octavia load balancers. You can enable this integration as a day 2 operation. However, there are a few known issues to be aware of:

* Only TCP traffic is supported.
* Floating IPs attached to your Octavia load balancers will not be deleted if you delete your cluster.
* If you have multiple external networks, you need to set `floating-network-id` to the one you want FIPs to be created in.
* `manage-security-groups` will not work for non-admin tenants.
```